crates/api_models/src/admin.rs

```diff
@@ -518,6 +518,12 @@ pub struct MerchantConnectorWebhookDetails {
     pub additional_secret: Option<Secret<String>>,
 }
 
+#[derive(Debug, Clone, Serialize, Deserialize, ToSchema)]
+pub struct MerchantConnectorInfo {
+    pub connector_label: String,
+    pub merchant_connector_id: String,
+}
+
```

Comment:
Why not move this `payments.rs`?

Reply:
coz all types for merchant connector details are declared in admin.rs